.github/workflows/dev_module_build.yml

```
@@ -26,7 +26,7 @@ env:
 on:
   workflow_dispatch:
   pull_request:
-    types: [opened, edited, synchronize]
+    types: [opened, synchronize]
```

```suggestion
    types: [opened, reopened, synchronize]
```

I don't know if we need "reopened" here. It is rare we reopen PRs.

---

I think you're right, we need this option.